### ISSUE TITLE
Question about ARM/Rpi support

### ISSUE THREAD

**Issue by dgbrewer1989** [akhilrex/podgrab#276]
I've attempted to install podgrab a few times today (linux manual install) on my raspberry pi 4 model b and noticed it was failing to build because it couldn't find the standard library. It appears you can toggle CGO_ENABLED and it'll build but it always fails to run.

Is this project supported on ARM devices?

Thank you!

**Comment by republicandaddy** [comment]
Did you ever find an answer to this?  I have a K3s Cluster I'd like to use for this.  It's a mixed cluster of X86 and Pi's but would prefer to set this one to run on Pi's only.

**Comment by dgbrewer1989** [comment]
> Did you ever find an answer to this?  I have a K3s Cluster I'd like to use for this.  It's a mixed cluster of X86 and Pi's but would prefer to set this one to run on Pi's only.

Nope, never found a solution sadly. I dropped the whole idea of a local podcast grabber for a while but coming back to it I'll probably just checkout https://github.com/SamTV12345/PodFetch since the author of that repo has popped into these issues off and on lately. 

Good luck!

**Comment by republicandaddy** [comment]
Thanks for the input.  I ended up going with AudioBookshelf.  It works well.  The only thing I miss from my prior one is the auto play list for new episodes.

**Comment by Sudneo** [comment]
Not sure if it helps, but I run this on my RPi k0s cluster without issue.

```
docker pull akhilrex/podgrab --platform linux/arm64
```

Worked fine for me (I have pushed back the image in my registry). You can choose platform on [Docker hub](https://hub.docker.com/layers/akhilrex/podgrab/latest/images/sha256-ad7ad074bbb77b039f77d238f9d56f4f7598e1645e801e26c9f06f48b0f2f879?context=explore).

Looking at the [Github action](https://github.com/akhilrex/podgrab/blob/44e2b1c207288bb8a84ecb64424e0a501fa02510/.github/workflows/hub.yml#L42C29-L42C29) I also don't see anything particular done to build it.
